### ISSUE TITLE
Quest: Survey the Land (mangos one)

### ISSUE THREAD

**Issue by Mibu86** [CoronaCore/Issues#107]
For some strange reason if u click the dragon without having the quest u get mounted and start  Flying as it is intended.
Issue is that this happens without having the quest....
And.. when you take quest ..it seems the dragon do-sent wanna fly u there anymore
It's one time Ride...so u should make it not accessible to players that don't have the quest or dunno
![wowscrnshot_071314_223126](https://cloud.githubusercontent.com/assets/6356589/3565615/63e22544-0ac4-11e4-9e46-aa87d5777886.jpg)
That's the dragon try it without quest..then take quest and try again (u wount be able to complete the quest)

**Comment by Salja** [comment]
Have the npc a gossip to fly ?

**Comment by Mibu86** [comment]
normally i believe it has..but this one do-sent have any...
The dragons in shatara have gossips..so this one should have two..
i just accidentally right clicked him...and found myself flying .. that's how i found the bug

**Comment by Salja** [comment]
yeah i see some mitchmatch in gossip for this npc i will fix tomorrw wehn germany wins
